### PR TITLE
Add Audio Input Level to Report Stat.

### PR DESCRIPTION
**Pull request by jbelmont**
Add Audio Level Stat

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "webrtc-stats-gatherer",
   "description": "Gathers stats on interval for webrtc peer connection",
-  "version": "2.1.0",
+  "version": "3.0.0",
```
Comment (xdumaine) [inline bower.json]: I think 2.2.0 is fine, since this is not a breaking change.

**Comment by xdumaine** [comment]
Approved with change from 3.0.0 to 2.2.0

File: bower.json (hunk 1 above)
Comment (jbelmont) [inline bower.json]: Out of curiosity why is web-directory at version 3.0.0 on bower.json

Comment (xdumaine) [inline bower.json]: That is a very good question.

**Pull request merged**